lib/Voxa-Opearlo.js

```diff
@@ -26,27 +26,25 @@ function register(skill, config) {
   skill.onAfterStateChanged((alexaEvent, reply, transition) => {
     OpearloAnalytics.registerVoiceEvent(
       alexaEvent.session.user.userId,
-      transition.to,
+      'Custom',
       alexaEvent.intent.name,
```

Comment:
the custom even type should probably be `stateTransition` or something. Also, not sure if we should be tracking the intent.params for state transition events